src/drake/shell.clj

```diff
@@ -96,7 +100,7 @@
         proc (.exec (Runtime/getRuntime)
                     cmd-for-exec
                     ^"[Ljava.lang.String;" env
-                    (fs/file fs/*cwd*))]
+                    (fs/file (get-run-dir exec-dir)))]
```

Comment:
sorry for the late, disorganized comment. considering how short `get-run-dir` is now, and that it's only called here, maybe just remove it and inline the code instead? i mean, it's almost the same amount code as calling it, heh